web/src/app/workplace/components/task-list-toolbar/task-list-toolbar.component.ts

```diff
@@ -54,21 +60,43 @@ export class TaskListToolbarComponent implements OnInit {
   ) {}
 
   ngOnInit() {
-    this.workbasketService.getAllWorkBaskets().subscribe((workbaskets) => {
-      this.workbaskets = workbaskets.workbaskets;
-      this.workbaskets.forEach((workbasket) => {
-        this.workbasketNames.push(workbasket.name);
+    this.workbasketService
+      .getAllWorkBaskets()
+      .pipe(takeUntil(this.destroy$))
+      .subscribe((workbaskets) => {
+        this.workbaskets = workbaskets.workbaskets;
+        this.workbaskets.forEach((workbasket) => {
+          this.workbasketNames.push(workbasket.name);
+        });
+
+        // get workbasket of current user
+        const user = this.taskanaEngineService.currentUserInfo;
```

Comment:
would it make sense to create an user state to store the user info? Would make sense to use the store for access ids as well

Reply:
Yes, I think this is a good idea. However, I‘d prefer to keep it as simple as it currently is for the next release and postpone the creation a new store.